bundle/tests/monitor/databricks.yml

```diff
@@ -0,0 +1,39 @@
+resources:
+  monitors:
+    my_monitor:
+      table_name: "main.test.thing1"
+      assets_dir: "/Shared/provider-test/databricks_monitoring/main.test.thing1"
+      output_schema_name: "test"
+      inference_log:
+        granularities: ["1 day"]
+        timestamp_col:  "timestamp"
+        prediction_col: "prediction"
+        model_id_col:  "model_id"
+        problem_type: "PROBLEM_TYPE_REGRESSION"
+
+
+targets:
+  development:
+    mode: development
+    resources:
+      monitors:
+        my_monitor:
+          table_name: "main.test.dev"
+  staging:
+    resources:
+      monitors:
+        my_monitor:
+          table_name: "main.test.staging"
+          output_schema_name: "staging"
+  production:
+    resources:
+      monitors:
+        my_monitor:
+          table_name: "main.test.prod"
+          output_schema_name: "prod"
+          inference_log:
+            granularities: ["1 hour"]
```

Comment:
Yes they are appended, for the user this would granularities for aggregating data into time windows. Should we add a comment in the documentation somewhere?

Reply:
Sure, you could do that. This is default DAB behavior.